### ISSUE TITLE
TextChannel.overwrites raises TypeError

### ISSUE THREAD

**Issue by TrixiS** [nextcord/nextcord#601]
### Summary

TextChannel.overwrites raises TypeError

### Reproduction Steps

Just use .overwrites property or .overwrites_for

### Minimal Reproducible Code

```python
for channel in ctx.guild.channels:
    channel_ow = channel.overwrites
```

### Expected Results

PermissionOverwrites object

### Actual Results

TypeError (see below)
![image](https://user-images.githubusercontent.com/55791844/163444133-b7839401-2117-4752-b443-2fb5821d2b9a.png)

### Intents

All

### System Information

- Python v3.8.9-final
- nextcord v2.0.0-alpha
    - nextcord pkg_resources: v2.0.0a3
- aiohttp v3.7.4.post0
- system info: Windows 10 10.0.22000

### Checklist

- [X] I have searched the open issues for duplicates.
- [X] I have shown the entire traceback, if possible.
- [X] I have removed my token from display, if visible.

**Comment by TAG-Epic** [comment]
Please send the full traceback as we cannot fully understand what the issue is from just a image of the first line of the error.

**Comment by DenverCoder1** [comment]
Upgrade nextcord. v2.0.0a3 is very old.

**Comment by TrixiS** [comment]
> Please send the full traceback as we cannot fully understand what the issue is from just a image of the first line of the error.

```
Traceback (most recent call last):
  File "c:\Users\shata\.virtualenvs\deloriangray_bot-3mRO6FTr\lib\site-packages\nextcord\ext\commands\core.py", line 168, in wrapped
    ret = await coro(*args, **kwargs)
  File "C:\Users\shata\source\kwork\deloriangray_bot\bot\cogs\settings.py", line 20, in set_role
    channel_ow = channel.overwrites
  File "c:\Users\shata\.virtualenvs\deloriangray_bot-3mRO6FTr\lib\site-packages\nextcord\abc.py", line 486, in overwrites
    overwrite = PermissionOverwrite.from_pair(allow, deny)
  File "c:\Users\shata\.virtualenvs\deloriangray_bot-3mRO6FTr\lib\site-packages\nextcord\permissions.py", line 714, in from_pair
    for key, value in allow:
  File "c:\Users\shata\.virtualenvs\deloriangray_bot-3mRO6FTr\lib\site-packages\nextcord\flags.py", line 134, in __iter__
    yield (name, self._has_flag(value.flag))
  File "c:\Users\shata\.virtualenvs\deloriangray_bot-3mRO6FTr\lib\site-packages\nextcord\flags.py", line 137, in _has_flag
    return (self.value & o) == o
TypeError: unsupported operand type(s) for &: 'int' and 'NoneType'
```



> Upgrade nextcord. v2.0.0a3 is very old.

It is the latest version from pypi, isn't it?

**Comment by TAG-Epic** [comment]
Latest from pypi is a10

**Comment by TAG-Epic** [comment]
Fixed in #219
